src/ui/components/CardsStack/CardsStack.tsx

```diff
@@ -65,7 +65,7 @@ const CardsStack = ({
     let pathname = "";
 
     if (cardsType === CardType.IDENTIFIERS) {
-      const data = cardsData[index] as DIDDetails;
+      const data = cardsData[index] as JSONObject;
```

Comment:
Won't this be `KERIDetails`? (or renamed to `IdentifierDetails` based on my last comment)